templates/createDropbox.hbs

```diff
@@ -91,6 +105,18 @@
       <button type="submit" id="submitDropbox" class="btn btn-block btn-lg btn-primary">Submit Documents</button>
     </div>
   </form>
+  <script type="text/javascript">
+    function validateIdentification() {
+      if (document.querySelector('#customerDob').value !== ''
+        || document.querySelector('#customerReference').value !== ''
+        || document.querySelector('#customerNationalInsurance').value !== '') {
+        return true;
+      } else {
+        document.querySelector('#identifiable-fields-warning').classList.remove('visually-hidden')
+        return false;
+      }
+    }
+  </script>
```

Comment:
Not sure on the original requirements, but are we cool with adding JS to this client-side - wasn't sure if there was a requirement for it to work without JS enabled? (I guess it still will work here, just the form won't be validated).

Reply:
Maybe one for @bjpirt or @mmmmillar?